src/main/java/com/spotify/docker/client/messages/Version.java

```diff
@@ -27,6 +27,7 @@
 
   @JsonProperty("ApiVersion") private String apiVersion;
   @JsonProperty("Arch") private String arch;
+  @JsonProperty("BuildTime") private String buildTime;
```

Comment:
Hi John, It will be  a string in RFC3339Nano  format. I added a new method
 parseRFC3339Nano()  in client/DockerDateFormat.java which we can use to parse this.